### PR TITLE
Added support for doctrine dbal 3

### LINKED ISSUE #256: Check for doctrine does not work with doctrine/dbal v3
We are trying to use the bundle with doctrine/dbal v3  and it does not work for the following reasons: 

```
Error: Call to undefined method Doctrine\DBAL\Connection::fetchColumn()
/builds/wizifarm/wizifarm/vendor/liip/monitor-bundle/Check/DoctrineDbal.php:24
```

According to https://www.doctrine-project.org/2020/11/17/dbal-3.0.0.html the method has been removed and we must use `fetchOne`.

I am ready to do a PR but I don't know how to properly handle dbal 2 and 3 at the same time :
I was thinking of using `Doctrine\DBAL\Version` but the class is marked as internal and deprecated.

### PR DESCRIPTION
**Pull request by notFloran**
Fix #256

### CODE REVIEW

File: Check/DoctrineDbal.php [hunk 1]
```diff
@@ -21,7 +21,12 @@ public function check()
     {
         $connection = $this->manager->getConnection($this->connectionName);
         $query = $connection->getDriver()->getDatabasePlatform()->getDummySelectSQL();
-        $connection->fetchColumn($query);
+
+        if (method_exists($connection, 'fetchOne')) {
```
Comment (kbond) [inline Check/DoctrineDbal.php:25]: Can you add a comment here to explain why this is needed?

**Review by kbond — CHANGES_REQUESTED** [review]

File: Check/DoctrineDbal.php (hunk 1 above)
Comment (notFloran) [reply Check/DoctrineDbal.php:26]: Done

**Review by kbond — APPROVED** [review]

**Comment by kbond** [comment]
Thanks @notFloran!

**Comment by kbond** [comment]
[Done](https://github.com/liip/LiipMonitorBundle/releases/tag/2.16.0).

**Pull request merged**